main/src/ca/uwaterloo/flix/language/phase/Lexer.scala

```diff
@@ -360,6 +360,7 @@ object Lexer {
       case _ if isKeyword("Map#") => TokenKind.MapHash
       case _ if isKeyword("List#") => TokenKind.ListHash
       case _ if isKeyword("Vector#") => TokenKind.VectorHash
+      case _ if isMatchPrev("d\"") => acceptDebugString()
```

Comment:
I can accept the single-letter prefix if this becomes a DebugString type. Otherwise it needs to be uglier to discourage abuse.